src/web/MarkdownTextInput.css

```diff
@@ -30,11 +30,34 @@
   padding-right: 1px !important;
 }
 
-.react-native-live-markdown-input-singleline:empty::before,
-.react-native-live-markdown-input-multiline:empty::before {
-  pointer-events: none;
-  display: block; /* For Firefox */
-  content: attr(placeholder);
```

Comment:
@BartoszGrajdek Why do we removed this part? It seems to cause an issue of not showing the placeholder

https://expensify.slack.com/archives/C049HHMV9SM/p1755884247194659